navapbc/platform-test
# Rocket/test 474

## Ticket

N/A

## Changes

> What was added, updated, or removed in this PR.

- Add a test docker image specifically for testing the changes in 474

## Context for reviewers

> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers.

This PR is for testing the changes proposed in https://github.com/navapbc/template-infra/pull/474.

## Testing

> Provide evidence that the code works as expected. Explain what was done for testing and the results of the test plan. Include screenshots, [GIF demos](https://www.cockos.com/licecap/), shell commands or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

This PR changes `/infra/app/services/main.tf` to use a different configuration that exercises the functionality added in 474. Specifically, it uses a separate docker image (https://hub.docker.com/r/rocketnovadockerhub/tiny-env-test) which should (if everything is set up correctly) show something like this:

```
Hello, World!

Custom env var: 100

Custom secret: 200
```

Testing steps:

1. Manually create a secret in Parameter Store with the path `/custom_secret`. The value can be anything. In the example above, I arbitrarily set it to `200`.
2. Deploy these changes. I used `make infra-update-app-service APP_NAME=app ENVIRONMENT=dev`
3. Navigate to the ECS task's LB url. It should display something like the above (showing whatever value you set the Parameter Store secret to)
